README.md

```diff
@@ -41,143 +44,45 @@ export const trpcMsw = createTRPCMsw<AppRouter>() /* 👈 */
 
 ```typescript
 const server = setupServer(
-  trpcMsw.userById.query((req, res, ctx) => {
-    return res(ctx.status(200), ctx.data({ id: '1', name: 'Uncle bob' }))
-  }),
-  trpcMsw.createUser.mutation(async (req, res, ctx) => {
-    return res(ctx.status(200), ctx.data({ id: '2', name: await req.json() }))
-  })
+  trpcMsw.userById.query(() => ({ id: '1', name: 'Uncle bob' })),
+  trpcMsw.createUser.mutation((name) => ({ id: '2', name }))
 )
 ```
 
+You can find examples of how to use it in the [test-react](./packages/test-react/package.json) package or in the [test-node](./packages/test-node/package.json) package.
+
 ## How it works
 
 `createTRPCMsw` returns a Proxy that infers types from your AppRouter
 
 ```typescript
 // all queries will expose a query function that accepts a MSW handler
-trpcMsw.myQuery.query((req, res, ctx) => {})
+trpcMsw.myQuery.query(() => {})
 
 // all mutations will expose a mutation function that accepts a MSW handler
-trpcMsw.myMutation.mutation((req, res, ctx) => {})
-```
-
-**supports merged routers**
-
-```typescript
-// @filename: routers/_app.ts
-// taken from https://trpc.io/docs/merging-routers
-import { userRouter } from './user'
-import { postRouter } from './post'
-
-const appRouter = router({
-  user: userRouter, // put procedures under "user" namespace
-  post: postRouter, // put procedures under "post" namespace
-})
-
-// @filename: frontend/test/PostList.tsx
-
-// all nested routers will be infered properly
-trpcMsw.user.list.query((req, res, ctx) => {})
-```
-
-## MSW Augments
-
-**ctx.data**
-
-Inspired by MSW GraphQL the context of your handlers now exposes a data function that will transform your data to the tRPC structure
-
-```typescript
-mswTrpc.userById.query((req, res, ctx) => {
-  console.log(ctx.data({ my: 'data' })) /* 👈 */
-})
-
-// return ctx.json with
-{
-  result: {
-    data: {
-      my: 'data'
-    }
-  }
-}
+trpcMsw.myMutation.mutation(() => {})
 ```
 
-**req.getInput**
-
-Returns the parsed input from the request
-
-```typescript
-//router.ts
-const appRouter = t.router({
-  userById: t.procedure.input(z.object({ name: z.string() })).query(req => {
-    const { input } = req
-    const user = userList.find(u => u.name === input.name)
-    return user
-  }),
-})
-
-//test-file.ts
-mswTrpc.userById.query((req, res, ctx) => {
-  console.log(req.getInput()) /* 👈 */
-})
-
-mswTrpc.createUser.mutation(async (req, res, ctx) => {
-  console.log(await req.getInput()) /* 👈 in mutation handle getInput returns a promise because it uses req.json() */
-})
-
-// outputs
-{
-  name: 'Pedro'
-}
-
-test('should do something', () => {
-  trpc.userById.query({ name: 'Pedro' })
-})
-```
-
-Please note:
-
-- calling `req.getInput` and req.json in the same mutation handler will fail
-
 ## Config
 
-`createTRPCMsw` accepts a 2nd argument:
+You need to pass a `httpLink` to the `createTRPCMsw` function like you would do with the tRPC client.
```

Comment:
This description must come before the `**2. build your trpcMsw with createTRPCMsw.**` in the `## Usage` section  above since the `config` argument of `createTRPCMsw` is now non-optional.